### PR TITLE
CSI 4731 servicer create vg implementation

### PR DESCRIPTION
**Pull request by ArbelNathan**

### CODE REVIEW

File: controllers/tests/controller_server/common.py [hunk 1]
```diff
@@ -17,6 +17,15 @@ def mock_array_type(contex, server_path):
     contex.addCleanup(detect_array_type_patcher.stop)
 
 
+def mock_utils_method(contex, utils_method_name, return_value=None):
+    utils_method_path = '.'.join(('controllers.servers.utils', utils_method_name))
```
Comment (matancarmeli7) [inline controllers/tests/controller_server/common.py:21]: can you move `controllers.servers.utils` to test_settings?

Comment (ArbelNathan) [reply controllers/tests/controller_server/common.py:21]: settings is mainly for consts..

**Review by ELENAGER — APPROVED** [review]
LGTM

File: controllers/tests/controller_server/common.py (hunk 1 above)
Comment (matancarmeli7) [reply controllers/tests/controller_server/common.py:21]: I think it's also a const, but ok, it doesn't necessary

**Review by matancarmeli7 — APPROVED** [review]
LGTM

**Pull request merged**